azavea/nasa-hyperspectral
# Add Aviris NG Collection to Aviris Catalog

## Overview

The Aviris catalog now looks like:

- Catalog: "aviris"
  - Collection: "aviris-classic"
  - Collection: "aviris-ng"

Closes #79, Closes #93

## Notes

I refactored the one file script out to a few different files. Hopefully that keeps things a little be more logically separated. 

This PR is likely best digested by individual commit. The changes necessary to ingest NG are in the first commit. The refactor which does not affect functionality is in the second two.

## Testing Instructions

Run `./scripts/update` to ensure the catalogs-aviris container is up to date.

`docker-compose run --rm aviris-catalog` should now generate a catalog with both sub collections. This catalog should continue to successfully ingest into franklin.

## Checklist
